keyboards/handwired/twadlee/tp69/readme.md

```diff
@@ -0,0 +1,17 @@
+# Twadlee TP69
+
+![tp69](https://i.imgur.com/sC1qmJS.png)
+69-key keyboard having a Trackpoint unit under an additional 70th keycap.
```

Comment:
Adding another line break here improves the readability/visibility of the description.

```suggestion

A 69-key keyboard having a Trackpoint unit under an additional 70th keycap.
```